packages/vscode-apollo/src/testRunner/jest-config.ts

```diff
@@ -0,0 +1,20 @@
+import { resolve } from "path";
```

Comment:
Is this file something we could hoist to the root, to make it more obvious where to find it?

Reply:
I'm tempted to leave it here next to its related files, since it resolves the environment and setup framework scripts which make up the whole testing suite. I'm open to other viewpoints - but this isn't meant to be consumed in the way that running jest normally would since it's imported and passed into the `jest.runCLI` function.